setup.py

```diff
@@ -124,7 +124,7 @@
                       'tests', 'tests.*',
                       '*.tests', '*.tests.*')),
     zip_safe=False,
-    include_package_data=True,
+    include_package_data=False,
```

Comment:
This does not remove the libraries and so on from wheels, because those are explicitly included by `package_data`, though I'm not sure how this interacts with `sdist`. I'll have to test that. I suspect the generated C source files might need to be included via MANIFEST.

Reply:
Oops, turns out we already have a MANIFEST.in so this is a non-issue and works as-is.